boards/arm/xilinx_kv260_r5/doc/index.rst

```diff
@@ -0,0 +1,139 @@
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| GIC          | on-chip    | generic interrupt    |
+|              |            | controller           |
++--------------+------------+----------------------+
+| TTC          | on-chip    | system timer         |
++--------------+------------+----------------------+
+| UART         | on-chip    | serial port          |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Timer
+------------
+
+This board configuration uses a system timer tick frequency of 1000 Hz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+on-chip UART1.
+
+Memories
+--------
+
+Although Flash, DDR and OCM memory regions are defined in the DTS file,
+all the code plus data of the application will be loaded in the sram0 region,
+which points to the DDR memory. The ocm0 memory area is currently available
+for usage, although nothing is placed there by default.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Dual-redundant Core Lock-step (DCLS) execution is not supported yet.
+* Only the first core of the R5 subsystem is supported.
+* Xilinx Zynq TTC driver does not support tickless mode operation.
+* The Cortex-R5 and the Cortex-A53 shares the same UART controller, more details below.
+
+Programming and Debugging
+*************************
+
+Currently the best way to run this sample is by loading it through remoteproc
+from the APU, running Linux, to the RPU, assuming the target board has a compatible
+Linux kernel (Versions 5.15 from Xilinx, 6.0 and 6.2 are supported). Users can make use of
+Xilinx's pre-built Petalinux reference images as a starting point to enable
+remoteproc support, as described here:
+
+https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/1641152513/Kria+K26+SOM#PetaLinux
+
+The above link includes instructions on how to prepare a SD-Card to run this image on
+the target board.
+
+After getting the Linux image running on the target board, build a Zephyr application,
+such as the hello world sample shown below:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :host-os: unix
+   :board: xilinx_kv260_r5
+   :goals: build
+
+Due to a hardware limitation, both Linux and the Zephyr share the same
+UART controller, then when the Zephyr application gets launched it will takeover the
+console from Linux, to overcome this limitation the best way is to connect to the board
+using the ``ssh`` over the network, then connect the board and the host computer in the
+same network, assuming the usage of the reference image from Xilinx open a terminal then:
```

Comment:
I already approved, so more of a nit for clarity, but this might be easier to follow:

```suggestion
Due to a hardware limitation, both Linux and Zephyr share the same UART
controller, meaning when the Zephyr application is started it will takeover the
console from Linux.

To avoid this limitation when accessing the Linux shell, the best approach is to
connect to the board using ``ssh`` over the network (not using the FTDI
USB interface on the board), with the dev board and the host computer
connected to the same network. 

Assuming you are using the default ``petalinux`` user from the Xilinx
reference image , open a terminal on the host machine and ssh into the
development board with the boards IP address (found via ``ifconfig``):
```